server/lib/nl_page_config.py

```diff
@@ -350,6 +350,39 @@ def build_page_config(detection: Detection, data_spec: DataSpec):
       category.stat_var_spec[sv_key].name = sv2name[primary_sv]
       category.stat_var_spec[sv_key].denom = "Count_Person"
 
+  # Render scatter plot if query asks for a correlation
+  # IMPORTANT: Right now this is fragile, as it will only
+  # use SVs from the most recent context.
+  # Also assumes that
+  #    - previous context had a primary_sv that was not "".
```

Comment:
if previous context does not have a primary_sv, we should walk up the history chain to find one. there could be null contexts in between. for later also.